src/main/java/org/beanpod/switchboard/dto/mapper/LogMapper.java

```diff
@@ -11,4 +14,17 @@
   List<LogModel> toLogModels(List<LogEntity> logEntityList);
 
   LogModel toLogModel(LogEntity logEntity);
+
+  LogEntity toLogEntity(LogDto logDto);
+
+  LogDto toLogDto(CreateLogRequest createLogRequest);
+
+  LogDto toLogDto1(LogEntity logEntity);
```

Comment:
let's rename this to something like `logEntityToDto

Reply:
Thank you for the suggestion, I have updated the method signature.